frontend/src/app/main/multi-select/multi-select.component.css

```diff
@@ -0,0 +1,79 @@
+.container {
```

Comment:
I seem to recall a mention that this was referenced from somewhere, should we include a link to credit it?

Reply:
My bad. This component is actually my own from another project. I will delete it since I did not end up using it and imported the multi-select component from the material library.